# LDAP Provider
vfs.provider.ldap/connect.error=Could not connect to LDAP server at "{0}".

